app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.kt

```diff
@@ -137,16 +61,43 @@ class ThirdPartyTransferFragment : BaseFragment(), ThirdPartyTransferView, OnIte
             savedInstanceState: Bundle?
     ): View? {
         (activity as BaseActivity?)?.activityComponent?.inject(this)
-        rootView = inflater.inflate(R.layout.fragment_third_party_transfer, container, false)
+        errorLayoutBinding = ErrorLayoutBinding.inflate(inflater,container,false)
+        _binding = FragmentThirdPartyTransferBinding.inflate(inflater,container,false)
         setToolbarTitle(getString(R.string.third_party_transfer))
-        if (rootView != null ) ButterKnife.bind(this, rootView!!)
-        sweetUIErrorHandler = SweetUIErrorHandler(activity, rootView)
+        sweetUIErrorHandler = SweetUIErrorHandler(activity, binding.root)
         showUserInterface()
         presenter?.attachView(this)
         if (savedInstanceState == null) {
             presenter?.loadTransferTemplate()
         }
-        return rootView
+
+        binding.btnReviewTransfer.setOnClickListener {
```

Comment:
We can put all of these view interactions(listeners) inside onViewCreated() this way we can be sure that the view has already been created and now we can interact with the individual view elements.